README.md

```diff
@@ -497,9 +497,9 @@ Kogito operator integrates with various other technologies and operators. The te
 | Technology         | Tested version                                                |
 | ------------------ | ------------------------------------------------------------- |
 | Infinispan         | Infinispan operator 2.1.4 (deployed by OLM `2.1.x` channel)   |
-| Kafka              | Strimzi 0.23.0 (deployed by OLM `stable` channel)             |
-| Keycloak           | Keycloak operator 12.0.3 (deployed by OLM `alpha` channel)    |
+| Kafka              | Strimzi 0.24.0 (deployed by OLM `stable` channel)             |
+| Keycloak           | Keycloak operator 14.0.0 (deployed by OLM `alpha` channel)    |
 | Prometheus         | Prometheus operator 0.47.0 (deployed by OLM `beta` channel)   |
-| Grafana            | Grafana operator 3.10.1 (deployed by OLM `alpha` channel)     |
+| Grafana            | Grafana operator 3.10.3 (deployed by OLM `alpha` channel)     |
```

Comment:
I think we need to add Knative here as well. See: https://issues.redhat.com/browse/KOGITO-5552

Reply:
Added, from my local test it seems that 0.22 and 0.23 don't work properly for some reason. Will keep the tests on 0.20 until we migrate to the newest version.

Reply:
Weird... I have an env with KogitoSource + KogitoOperator on 0.22.. but you're right, maybe there's something different. As soon as we add everything to the Knative side, we can remove this integration from the operator. Let's keep on 0.20.0